src/hexagon/testing/setupTests.ts

```diff
@@ -29,9 +36,16 @@ beforeEach(() => {
 
 // Reset all mocks after each test
 afterEach(() => {
+  // Reset the adapter mocks to their default implementations
+  resetMockVocabularyAdapter();
+  resetMockSubcategoryAdapter();
```

Comment:
beforeEach has a wrapper function abstracting what adapters are mocked, the same pattern should be used here in afterEach (like a resetAdapterMocks function called here)
```suggestion
  resetMockSubcategoryAdapter();
```

Reply:
Done